File: Filas y Columnas/Columnas.js
const r=3;
const c=3;

let TwoDArray=[[1,2,3],[4,5,6],[7,8,9]];



console.log("Los elementos del array bidimensional son: ");
for(let x=0;x<r;x++){
    let fila =" ";
    for(let y=0;y<c;y++){
        fila+=TwoDArray[x][y]+ " ";
    }
    console.log(fila);
}
console.log("\nLos elementos del array bidimensional son: ");
for(let y=0;y<c;y++){
    let resultado =" ";
    for(let x=0;x<r;x++){
        resultado+=TwoDArray[x][y]+ " ";
    }
    console.log(resultado);
}

File: Filas y Columnas/Columnas.py
r=3;c=3
arr=[0]*r*c
TwoDArr=[[1,2,3],[4,5,6],[7,8,9]];
k=0
for y in range(c): #inicio con las columnas y luego con las filas
    for x in range(r):
        k=y*r+x; #la formula cambia 
        arr[k]=TwoDArr[x][y]
        k=k+1
print("Los elementos del array bidimensional son: ")
for row in TwoDArr:
    for ele in row:
        print(ele,end=" ")
    print()
print("\nLos elementos del array unidimensional son: ")
for x in range(len(arr)):
    print(arr[x],end=" ")
